src/components/cards/CardCvcInput/hook/useCardCvcInput.ts

```diff
@@ -0,0 +1,33 @@
+import { ChangeEvent, useCallback } from "react";
+
+import { isNumber } from "@/components/helper";
+import { CVC_NUMBER_MAX_LENGTH } from "@/constants/variables";
+import useInput from "@/hooks/useInput";
+
+const useCardCvcInput = (initialValue: string) => {
+  const { value, onChange } = useInput(initialValue);
+
+  const handleCvcNumberChange = useCallback(
+    (e: ChangeEvent<HTMLInputElement>) => {
+      const { target } = e;
+
+      if (!(target instanceof HTMLInputElement)) return;
+
+      console.log(isNumber(target.value));
+
+      if (target.value && !isNumber(target.value)) return;
+
+      if (target.value.length > CVC_NUMBER_MAX_LENGTH) return;
```

Comment:
```suggestion
      if (validateCVCNumber(event.target.value) === false) return;
```

이렇게 추상화하면 읽어내려가는데 더 자연스러울 것 같아요.
어떤 valdiation이 필요한지 다른 함수로 분리해봅시다 :)